test/jdk/sun/security/tools/jarsigner/RemovedFiles.java

```diff
@@ -40,8 +40,15 @@ public class RemovedFiles {
 
     private static final String NONEXISTENT_ENTRIES_FOUND
             = "This jar contains signed entries for files that do not exist. See the -verbose output for more details.";
+    private static final String WEAK_UNSIGNED
+            = "The jar will be treated as unsigned, because it is signed with a weak algorithm that is now disabled";
```

Comment:
I know the existing test doesn't do this, but would it make sense to read the `resources/jarsigner.properties` files to get these messages instead of hard-coding them here?  Seems that if anyone were to change the messages down the road, they'll likely find out during test.

Otherwise, change looks good to me.

Reply:
That's possible, but it would require users to cross-reference the properties file to find out the actual text. Also, the keys in the properties file are quite inconsistent now. Older ones are always the English message with non-alphanumeric characters replaced by dots, and newer ones may be of shorter title-style.

Yes, if the message is updated, test will need to be updated as well.